### ISSUE TITLE
[FEATURE]: Add Light Mode

### ISSUE THREAD

**Issue by Shiva953** [adityaoberai/Alt-Text-Generator#17]
**Describe the solution you'd like**
A light mode that inverts bg and text.

**Describe alternatives you've considered**

**Any other relevant issues that are present?**

**Comment by adityaoberai** [comment]
@Shiva953 there is a light mode present. However, currently, the web app changes theme depending on your system default.

Is this something you would like to work on?

**Comment by KreativKrish** [comment]
We should add a button to toggle from dark to light mode or vice-versa depending on the user preference @adityaoberai .

**Comment by adityaoberai** [comment]
@KreativKrish is this something you'd like to work on?

**Comment by KreativKrish** [comment]
> @KreativKrish is this something you'd like to work on?

Yes, I would like to work on it. Please assign it to me. @adityaoberai
